src/advanced/configuration-updater.md

```diff
@@ -333,11 +340,11 @@ with a state change if all of the following conditions take place:
   to the same configuration hash
 
 If all the checks pass, the execution results in modifying the `config_proposes`
```

Comment:
Off-topic: IMO, this explanation is unhelpful. I'd suggest to replace it with something like

> If all the checks pass, the proposal is recorded as a candidate for the next configuration. The validators can then vote for or against the proposal.

Same for the corresponding descriptions for `Vote` and `VoteAgainst` transactions.